pkg/glusterblock/driver.go

```diff
@@ -0,0 +1,71 @@
+package glusterblock
+
+import (
+	"github.com/gluster/gluster-csi-driver/pkg/glusterblock/config"
+
+	"github.com/gluster/glusterd2/pkg/restclient"
+	"github.com/golang/glog"
+	"github.com/kubernetes-csi/drivers/pkg/csi-common"
+)
+
+const (
+	glusterBlockCSIDriverName    = "org.gluster.glusterblock"
+	glusterBlockCSIDriverVersion = "0.0.9"
```

Comment:
glusterblock or glusterloopback?

driver version needs to be updated to v1.0.0

Reply:
Done